op-bindings/bindgen/remote_handlers.go

```diff
@@ -0,0 +1,282 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"text/template"
+
+	"github.com/ethereum-optimism/optimism/op-bindings/etherscan"
+)
+
+type contractData struct {
+	abi          string
+	deployedBin  string
+	deploymentTx etherscan.TxInfo
+}
+
+func (generator *bindGenGeneratorRemote) standardHandler(contractMetadata *remoteContractMetadata) error {
+	fetchedData, err := generator.fetchContractData(contractMetadata.Verified, "eth", contractMetadata.Deployments["eth"])
+	if err != nil {
+		return err
+	}
+
+	contractMetadata.DeployedBin = fetchedData.deployedBin
+
+	// If ABI was explicitly provided by config, don't overwrite
+	if contractMetadata.Abi == "" {
+		contractMetadata.Abi = fetchedData.abi
+	} else if fetchedData.abi != "" && contractMetadata.Abi != fetchedData.abi {
+		generator.logger.Warn("The given contract ABI differs from what was fetched from Etherscan", "contract", contractMetadata.Name)
+		generator.logger.Debug("ABIs", "given", contractMetadata.Abi, "fetched", fetchedData.abi)
+	}
+
+	if contractMetadata.InitBin, err = generator.removeDeploymentSalt(fetchedData.deploymentTx.Input, contractMetadata.DeploymentSalt); err != nil {
+		return err
+	}
+
+	// We're not comparing the bytecode for Create2Deployer with deployment on OP,
+	// because we're predeploying a modified version of Create2Deployer that has not yet been
+	// deployed to OP.
+	// For context: https://github.com/ethereum-optimism/op-geth/pull/126
+	if contractMetadata.Name != "Create2Deployer" {
```

Comment:
Maybe we should make the `create2deployer` method completely separate to avoid branching logic in an otherwise-standard handler?